### DISCUSSION TITLE
I have lost the admin link, how should I retrieve it? لینک ادمین را گم کردم، چطور می‌تونم بازیابیش کنم؟

### DISCUSSION THREAD

**Discussion by TheTraxex** [hiddify/Hiddify-Manager#120] [Q&A سوالات رایج] [answered]
لینک ادمین را گم کردم، چطور می‌تونم بازیابیش کنم؟

**Comment by hiddify-com** [comment]
باید حتما دسترسی به ssh داشته باشید
بعد از اون دستور زیر را اجرا کنید

```
/opt/hiddify-config/apply_configs.sh
```

**Comment by just1random** [comment, marked as answer]
**فارسی:**
این اتفاق فقط در صورتی امکان پذیر است که همچنان دسترسی به سرور را از طریق `SSH` داشته باشید.
در سرور دستورات زیر را اجرا کرده و منتظر بمانید تا به پایان برسد. لینک های ادمین شما در انتها نمایش داده خواهد شد.

```
cd /opt/hiddify-config/
./apply_configs.sh
```

**English:**
This is only possible if you still have `SSH` access to the server.
Use the following commands. In the end your admin links will be visible.

```
cd /opt/hiddify-config/
./apply_configs.sh
```

**Reply by 3mehdi** [reply]
سلام من خطای زیر میگیرم و لینکی بهم نمایش داده نمیشه , لینوکس من  Debian هست 

ERROR!!!! There is an error in the installation of python panel. Exit....
